doc/basics.md

```diff
@@ -394,6 +396,28 @@ int main(void) {
 }
 ```
 
+The following is a similar example where one first accesses
+an array matching the "statuses" key. The result is expected to be
+an array. One select the first value in the array. The result
+is expected to be an object. One then selects the vallue corresponding
+to the key "id". We expec the value to be a non-negative integer.
+
+```C++
+#include "simdjson.h"
+
+int main(void) {
+  simdjson::dom::parser parser;
+  simdjson::dom::element tweets;
```

Comment:
I think you can actually just do `auto tweets = parser.load("twitter.json");` and skip these two lines. Uses chaining to make it more compact, possibly easier to read.

Reply:
(And skip the line above.)

Reply:
I think that what you are proposing would throw, wouldn't it? I don't mind adding a *new* example where we follow what you suggest,  but I think it has to appear elsewhere because in this context, we are discussing the exception-free mode.

Reply:
I added another example, but in a different section.